### PR TITLE
Publish @slack/web-api@6.7.1

### PR DESCRIPTION
**Pull request by seratch**
###  Summary

Releasing a new version soon...

* https://github.com/slackapi/node-slack-sdk/milestone/40?closed=1
* https://github.com/slackapi/node-slack-sdk/compare/@slack/web-api@6.7.0...39478435dbe3917a44c92906d0989852342eb084

### Requirements (place an `x` in each `[ ]`)

* [x] I've read and understood the [Contributing Guidelines](https://github.com/slackapi/node-slack-sdk/blob/master/.github/contributing.md) and have done my best effort to follow them.
* [x] I've read and agree to the [Code of Conduct](https://slackhq.github.io/code-of-conduct).

### CODE REVIEW

**Review by filmaj — COMMENTED** [review]
Should this release be a minor or is it scoped to a semver patch release? I don't see any new features that would warrant a minor release? Am I mistaken?

**Comment by seratch** [comment]
@filmaj 
>Should this release be a minor or is it scoped to a semver patch release? I don't see any new features that would warrant a minor release? Am I mistaken?

I think that a patch version can be fine too. All the changes are API response type additions.

File: packages/web-api/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@slack/web-api",
-  "version": "6.7.0",
+  "version": "6.8.0",
```
Comment (seratch) [inline packages/web-api/package.json:3]:
```suggestion
  "version": "6.7.1",
```

**Comment by seratch** [comment]
Changed the version to 6.7.1

**Review by mwbrooks — APPROVED** [review]
Added the patch label as well :)

**Pull request merged**